.travis.yml

```diff
@@ -24,9 +24,8 @@ jobs:
             install:
                 - mysql -e 'create database redaxo_5_0;'
                 - php redaxo/src/addons/tests/bin/setup.php
-            script:
-                - if find . -name "*.php" ! -path "*/vendor/*" -exec php -l {} 2>&1 \; | grep "syntax error, unexpected"; then exit 1; fi
-                - php redaxo/src/addons/tests/bin/run_tests.php
+            before_script: if find . -name "*.php" ! -path "*/vendor/*" -exec php -l {} 2>&1 \; | grep "syntax error, unexpected"; then exit 1; fi
```

Comment:
Vllt nur im codestyle build step linten?

Reply:
Vllt könnten man das linten auch paralellisieren

Reply:
Ich denke am wichtigsten ist es, in der niedrigsten PHP-Version zu linten.
Also vielleicht nur in 5.5?

Reply:
Habs jetzt als eigenen Build gemacht, wird parallel zu dem CS check ausgeführt.
Was meinste?

Reply:
Ich denke die anzahl der travis jobs muss auch so nierig sein wies geht.. ggf. Lint und coding standard beides auf 5.5 in einem job?

Reply:
Bin ich mir nicht sicher, was schneller ist. Eigene Builds, somit einer mehr, dafür aber parallel.
Vermutlich ist es mal so mal so.
Der CS-Fixer ist mit PHP 7 glaube ich deutlich schneller.